src/Main/ViewPlanesToggle.jsx

```diff
@@ -1,16 +1,17 @@
 import React from 'react'
-import { useActor } from '@xstate/react'
+import { useSelector } from '@xstate/react'
 import { viewPlanesIconDataUri } from 'itk-viewer-icons'
 import Button from 'react-bootstrap/Button'
 import Image from 'react-bootstrap/Image'
 import OverlayTrigger from 'react-bootstrap/OverlayTrigger'
 import Tooltip from 'react-bootstrap/Tooltip'
-import cn from 'classnames'
 
-function ViewPlanesToggle(props) {
+const ViewPlanesToggle = React.memo(function ViewPlanesToggle(props) {
   const { service } = props
-  const [state, send] = useActor(service)
-  const { slicingPlanes } = state.context.main
+  const selectCount = (state) => state.context.main
+  const stateSlicingPlanes = useSelector(service, selectCount)
```

Comment:
Can also do compress this to:
const stateSlicingPlanes = useSelector(service, (state) => state.context.main)

Reply:
Good point. I deleted selectCount in all of them.